core/src/main/scala/org/http4s/parser/AdditionalRules.scala

```diff
@@ -205,3 +206,10 @@ private[http4s] trait AdditionalRules extends Rfc2616BasicRules { this: Parser =
     }
   }
 }
+
+private[http4s] object AdditionalRules {
+  import Rfc2616BasicRules._
+  def EOI = P.char('\uFFFF')
```

Comment:
I put cats-parse based parsers here as temporary place, until AdditionalRules trait is fully migrated.